### PR TITLE
SFT-139 and SFT-140 - Add Aux BMS Fields to CCS

### PR DESCRIPTION
**Pull request by bill-luu**
Branch name incorrect, I got SFT 140 & 141 mixed up :/

### CODE REVIEW

File: EpsilonCCS/Inc/Data/AuxBmsData.h [hunk 1]
```diff
@@ -23,14 +40,39 @@ enum BatteryPrechargeState
...
+    unsigned char chargeTripDueToHighCellVoltage;
+    unsigned char chargeTripDueToHighTemperatureAndCurrent;
+    unsigned char chargeTripDueToPackCurrent;
+    unsigned char dischargeTripDueToLowCellVoltage;
+    unsigned char dischargeTripDueToHighTemperatureAndCurrent;
+    unsigned char dischargeTripDueToPackCurrent;
+    unsigned char protectionTrip;
+};
+
+struct AuxContactor
```
Comment (badeniran) [inline EpsilonCCS/Inc/Data/AuxBmsData.h:54]: Can this be renamed to something like `AuxContactorError`? Just to make it clear that it relates to errors concerning the contactors

Comment (badeniran) [reply EpsilonCCS/Inc/Data/AuxBmsData.h:54]: Or so that it's clear that it's related to debug information

File: EpsilonCCS/Inc/Data/AuxBmsData.h [hunk 2]
```diff
@@ -10,8 +10,25 @@
 #define AUX_VOLTAGE_MASK (0xF8) // Last 5 bits
 #define STROBE_MASK (0x01) // First bit
 #define ALLOW_CHARGE_MASK (0X02) // Second bit
-#define CONTACTOR_ERROR_MASK (0X04) // Third bit
-#define HIGH_VOLTAGE_ENABLE_MASK (0X08) //Fourth bit
+#define HIGH_VOLTAGE_ENABLE_STATE_MASK (0X04) // Third bit
+#define ALLOW_DISCHARGE (0X08) //Fourth bit
```
Comment (badeniran) [inline EpsilonCCS/Inc/Data/AuxBmsData.h:14]: Append `_MASK`

File: EpsilonCCS/Inc/Data/AuxBmsData.h [hunk 3]
```diff
@@ -10,8 +10,25 @@
...
+#define ORION_CAN_RECEIVED_RECENTLY (0x10) //Fifth bit
```
Comment (badeniran) [inline EpsilonCCS/Inc/Data/AuxBmsData.h:15]: Same as above

File: EpsilonCCS/Inc/Data/AuxBmsData.h [hunk 4]
```diff
@@ -10,8 +10,25 @@
...
+#define CHARGE_OPEN_BUT_SHOULD_BE_CLOSED_MASK (0x02)
+#define DISCHARGE_OPEN_BUT_SHOULD_BE_CLOSED_MASK (0x04)
+
+#define CHARGE_TRIP_DUE_TO_HIGH_CELL_VOLTAGE_MASK (0x01)
+#define CHARGE_TRIP_DUE_TO_HIGH_TEMPERATURE_AND_CURRENT_MASK (0x02)
+#define CHARGE_TRIP_DUE_TO_PACK_CURRENT_MASK (0x04)
+#define DISCHARGE_TRIP_DUE_TO_LOW_CELL_VOLTAGE (0x08)
+#define DISCHARGE_TRIP_DUE_TO_HIGH_TEMPERATURE_AND_CURRENT (0x10)
+#define DISCHARGE_TRIP_DUE_TO_PACK_CURRENT (0x20)
+#define PROTECTION_TRIP (0x40)
```
Comment (badeniran) [inline EpsilonCCS/Inc/Data/AuxBmsData.h:31]: Same as above

File: EpsilonCCS/Inc/Data/AuxBmsData.h (hunk 1 above)
Comment (bill-luu) [reply EpsilonCCS/Inc/Data/AuxBmsData.h]: I renamed it to `ContactorDebugInfo` to match the protocol.

**Pull request merged**